apis/cluster/cluster.go

```diff
@@ -63,6 +64,7 @@ type Cluster struct {
 	// The pattern here says that from an arbitrary deployment, you should be able
 	// to bootstrap anything else you could need. We default to the kubicorn controller.
 	ControllerDeployment *appsv1beta2.Deployment `json:"controllerDeployment,omitempty"`
+	CloudManagerSecret   *v1.Secret              `json:"cloudManagerSecret,omitempty"`
```

Comment:
We could remove this field to something like `APITokenSecret`, so it's easier to understand what's stored here.